src/Microsoft.Sbom.Tool/Microsoft.Sbom.Tool.csproj

```diff
@@ -24,5 +24,6 @@
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" />
     <PackageReference Include="Microsoft.Extensions.Hosting" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" />
```

Comment:
It shouldn't be necessary to add `Microsoft.Build.Utilities.Core` to the standalone tool
```suggestion
```